docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_OpenTracing_SmallRye.adoc

```diff
@@ -0,0 +1,73 @@
+[[MicroProfile_OpenTracing SmallRye]]
+= MicroProfile OpenTracing Subsystem Configuration
+
+Support for https://microprofile.io/project/eclipse/microprofile-opentracing[Eclipse MicroProfile OpenTracing] is provided by
+ the _microprofile-opentracing-smallrye_ subsystem.
+
+[[required-extension]]
+== Required Extension
+
+This extension is included in the standard configurations included in the WildFly distribution.
+
+You can also add the extension to a configuration without it either by adding
+an `<extension module="org.wildfly.extension.microprofile.opentracing"/>`
+element to the xml or by using the following CLI operation:
+
+[source]
+----
+[standalone@localhost:9990 /] /extension=org.wildfly.extension.microprofile.opentracing:add
+{"outcome" => "success"}
+
+[standalone@localhost:9990 /] /subsystem=microprofile-opentracing:add
+{
+    "outcome" => "success",
+    "response-headers" => {
+        "operation-requires-reload" => true,
+        "process-state" => "reload-required"
+    }
+}
+----
+
+== Supported instrumentation libraries
+
+WildFly's MicroProfile OpenTracing subsystem implements MicroProfile 1.1, which includes support for tracing JAX-RS and
+CDI.
+
+Additionally, applications being deployed are able to provide their own tracers via the
+https://github.com/opentracing-contrib/java-tracerresolver[`TracerResolver`] facility. In that case, the default tracer
+will *not* be used.
+
+The subsystem ships with https://github.com/jaegertracing/jaeger-client-java[Jaeger Java Client] as the default tracer.
+Refer to that project's documentation for instructions on how to configure it. Note that the configuration is only
+possible by setting system properties or environment variables. If a more complex configuration is required, implement a
+`TracerResolver` that returns the `Tracer` with the desired state.
+
+WARNING: by default, the Jaeger Client Java has a probabilistic sampling strategy, set to 0.001, meaning that only
+approximately one in one thousand traces will be sampled. To sample every request, set the environment variable
+`JAEGER_SAMPLER_TYPE` to `const` and `JAEGER_SAMPLER_PARAM` to `1`.
+
```

Comment:
Please add a paragraph something like this:

WARNING: the supported configuration options for this Tech Preview feature, particularly those related to configuring the Jaeger Java Client tracer, may change in incompatible ways in a future release.